app/assets/stylesheets/application.css

```diff
@@ -13,7 +13,7 @@
  */
 
 .btn-sul-dlss {
-  background-color: #8c1515;
+  background-color: #00548F;
```

Comment:
Where did this color come from? I'm not seeing it in the identity guide. https://identity.stanford.edu/design-elements/color/accent-colors/